Xkcd-Comics/ViewModels/ComicsVM.swift

```diff
@@ -31,20 +26,16 @@ class ComicsVM: ObservableObject {
     
     func fetchComics() {
         guard let url = URL(string: "https://xkcd.com/info.0.json") else { return }
-        
```

Comment:
It's better to space the code out a bit. You can keep single empty rows to visually group blocks of related code, as long as it's just one empty line and not more.